docs/integrations_api.yml

```diff
@@ -252,8 +270,20 @@ definitions:
         type: string
         format: datetime
         description: Server-side timestamp of the request reception.
+    example:
+      application/json:
+        id: "291ae0e5956c69c2267489213df4459d19ed48a806603def19d417d004a4b67e"
+        device_identity: "{\"mac\":\"00:01:02:03:04:05\", \"sku\":\"My Device 1\", \"sn\":\"SN1234567890\"}"
+        key: "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzogVU7RGDilbsoUt/DdH\nVJvcepl0A5+xzGQ50cq1VE/Dyyy8Zp0jzRXCnnu9nu395mAFSZGotZVr+sWEpO3c\nyC3VmXdBZmXmQdZqbdD/GuixJOYfqta2ytbIUPRXFN7/I7sgzxnXWBYXYmObYvdP\nokP0mQanY+WKxp7Q16pt1RoqoAd0kmV39g13rFl35muSHbSBoAW3GBF3gO+mF5Ty\n1ddp/XcgLOsmvNNjY+2HOD5F/RX0fs07mWnbD7x+xz7KEKjF+H7ZpkqCwmwCXaf0\niyYyh1852rti3Afw4mDxuVSD7sd9ggvYMc0QHIpQNkD4YWOhNiE1AB0zH57VbUYG\nUwIDAQAB\n-----END PUBLIC KEY-----\n"
+        status: "pending"
+        attributes:
+          mac: "{\"mac\":\"00:01:02:03:04:05\"}"
+          sku: "My Device 1"
+          sn:  "SN1234567890"
+        request_time: "2016-10-03T16:58:51.639Z"
+
   Status:
-    description: Admission status of the device
+    description: Admission status of a device.
```

Comment:
In deviceauth docs you used phrase "the device", here is "a device", I think we should stick to one of those.